scaffolding-chef-inspec/tests/linux/user-a2-profile-dep/inspec.yml

```diff
@@ -0,0 +1,12 @@
+name: effortless-a2-profile-test
+title: Effortless Automate CIS profile Test
+maintainer: humans@habitat.sh
+copyright: Chef Software Inc.
+copyright_email: humans@habitat.sh
+license: Apache-2.0
+summary: Effortless Automate CIS profile Test
+version: 0.1.0
```

Comment:
Please remove all metadata from this file

`maintainer`
`copyright`
`copyright_email`
`license`
`summary`


Please make the version `1.0.0`